spec/dummy/db/migrate/20230718181431_add_duplicate_notification.rb

```diff
@@ -0,0 +1,7 @@
+class AddDuplicateNotification < ActiveRecord::Migration[7.0]
+  def change
+    return unless Rails.env.production? && SecurityRole.count.positive?
+
+    Notifications::PossibleDuplicateFoundNotification.create! security_roles: SecurityRole.all
```

Comment:
these are auto-created now on demand, no need for migration

Reply:
updated this